### ISSUE TITLE
can we show a warning when we have a packages.config and Packages.props 

### ISSUE THREAD

**Issue by jmecosta** [microsoft/MSBuildSdks#182]
@jeffkl is it possible to show a warning when a project has a packages.config and a packages.props is found at root of the project. it has a few times that builds have fail and takes me a while to figure out that a packages.config is still found in the project folder and breaks the all central package defintion

thanks

**Comment by jeffkl** [comment]
Yes anything is possible.  I'm not sure if this quite makes sense though as there are valid scenarios where you need packages.config.  For example, Azure Service Fabric (.sfproj) and C++ (.vcxproj) only support `packages.config`.

Perhaps if we limited by file extension it would work well?
* .csproj
* .vbproj
* .fsproj

Or do a whitelist of extensions that are allowed to have `packages.config`?
* .sfproj
* .vcxproj

I'm fine with either approach, seems like a great feature considering central package versioning is all about maintaining a healthy code base.

**Comment by jmecosta** [comment]
What about a third approach and let users supress that with a parameter
defined in the project or globally?

We have vcxproj and use nugets but we removed all packages.configs from the
projects foldets and we use only central targets, similar approach as
package.props but uglier 😅

**Comment by jeffkl** [comment]
That could work too, you set a property to be told if you placed a `packages.config` next to a project that really should be using `PackageReference` and another property that just blindly logs a warning if it comes across a `packages.config`.

**Comment by jmecosta** [comment]
Yep sounds good to me... thanks

**Comment by dfederm** [comment]
Closing since CPV is deprecated
